site/app/controllers/AuthenticationController.php

```diff
@@ -308,4 +310,191 @@ public function userSelection() {
 
         return new WebResponse(AuthenticationView::class, 'userSelection', $users);
     }
+
+    public function sendVerificationEmail(string $email, string $verification_code, string $user_id): void {
```

Comment:
We presumably don't intend to use this method outside this class, so this method should be marked `protected` or `private` instead.